intel-sgx/ias/src/verifier/crypto.rs

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) Fortanix, Inc.
+ *
```

Comment:
Nit: I think in other crates we use the `module_name/mod.rs` file structure instead of `module_name.rs` for modules with sub-modules. Holds for the `verifier` module as well.